### PR TITLE
feat(floating-label): Add max-width mixin

### PR DESCRIPTION
**Pull request by EstebanG23**

### CODE REVIEW

File: packages/mdc-floating-label/README.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ Mixin | Description
 `mdc-floating-label-shake-keyframes($modifier, $positionY, $positionX, $scale)` | Generates a CSS `@keyframes` at-rule for an invalid label shake. Used in conjunction with the `mdc-floating-label-shake-animation` mixin.
 `mdc-floating-label-shake-animation($modifier)` | Applies shake keyframe animation to label.
 `mdc-floating-label-float-position($positionY, $positionX, $scale)` | Sets position of label when floating.
+`mdc-floating-label-max-width($max-width))` | Sets the width of the label.
```
Comment (kfranqueiro) [inline packages/mdc-floating-label/README.md]: width -> max-width

File: packages/mdc-floating-label/_mixins.scss [hunk 2]
```diff
@@ -67,3 +67,9 @@
     animation: mdc-floating-label-shake-float-above-#{$modifier} 250ms 1;
   }
 }
+
+@mixin mdc-floating-label-max-width($max-width) {
+  .mdc-floating-label {
```
Comment (kfranqueiro) [inline packages/mdc-floating-label/_mixins.scss]: Does it really make sense for this selector to be baked into this mixin? Would this not ordinarily be applied to a selector that already references the floating label root element? (See also the ink/fill color mixins)

File: packages/mdc-floating-label/README.md (hunk 1 above)
Comment (EstebanG23) [reply packages/mdc-floating-label/README.md]: Done!

File: packages/mdc-floating-label/_mixins.scss (hunk 2 above)
Comment (EstebanG23) [reply packages/mdc-floating-label/_mixins.scss]: Updated

File: packages/mdc-floating-label/README.md [hunk 3]
```diff
@@ -79,6 +79,7 @@ Mixin | Description
 `mdc-floating-label-shake-keyframes($modifier, $positionY, $positionX, $scale)` | Generates a CSS `@keyframes` at-rule for an invalid label shake. Used in conjunction with the `mdc-floating-label-shake-animation` mixin.
 `mdc-floating-label-shake-animation($modifier)` | Applies shake keyframe animation to label.
 `mdc-floating-label-float-position($positionY, $positionX, $scale)` | Sets position of label when floating.
+`mdc-floating-label-max-width($max-width))` | Sets the max width of the label.
```
Comment (kfranqueiro) [inline packages/mdc-floating-label/README.md]: Extra )

Comment (EstebanG23) [reply packages/mdc-floating-label/README.md]: Got it!

**Pull request merged**